torch/csrc/utils/python_dispatch.cpp

```diff
@@ -29,16 +29,71 @@ torch::Library::Kind parseKind(const std::string& k) {
 
 c10::optional<c10::DispatchKey> parseDispatchKey(const std::string& k) {
   static std::unordered_map<std::string, c10::DispatchKey> key_map = {
+    {"", c10::DispatchKey::Undefined},
```

Comment:
Need to make it easier to keep this up to date. Consider moving this in to c10/core/DispatchKey.cpp